src/main/java/com/google/ehub/utility/UserRecommendationUtils.java

```diff
@@ -0,0 +1,53 @@
+package com.google.ehub.utility;
+
+import com.google.ehub.data.FavoriteItemDatastore;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Utility class that implements the algorithm used to recommend the users that have the most common
+ * number of shared likes.
+ */
+public final class UserRecommendationUtils {
+  private static final FavoriteItemDatastore favoriteItemDatastore =
+      FavoriteItemDatastore.getInstance();
+
+  /**
+   * Finds the most recommended emails up to a given limit.
+   *
+   * @param userEmail the email of the user that wants to search for its recommended email list
+   * @param maxNumberOfRecommendations the maximum desired size for the recommendation list
+   * @return list containing the most recommended emails
+   */
+  public static List<String> getRecommendedEmails(
+      String userEmail, int maxNumberOfRecommendations) {
```

Comment:
You could put your MAX constant inside this class since right now there are no plans for anyone to call this method with different values of max

Reply:
OK I moved the constant inside the class since I am not going to be using this method elsewhere.